### DISCUSSION TITLE
Clone a previous retro

### DISCUSSION THREAD

**Discussion by jasondduncan** [microsoft/vsts-extension-retrospectives#551] [Ideas]
If creating custom retro templates is not possible, then it would be nice if we could clone our previous retro to create the new one without having to go update every column each time. Maybe add a menu item "Clone retrospective" under "Create new retrospective".

![image](https://github.com/microsoft/vsts-extension-retrospectives/assets/1136327/8ff19398-53dd-4326-aae2-3fbfa74628d1)

**Comment by JStuve** [comment]
I currently am working on this solution and will submit the PR once my current PR is merged.

I did plan on exploring what it'll take to implement template saving. 

[Open PR for duplicating](https://github.com/microsoft/vsts-extension-retrospectives/pull/561)

**Comment by JStuve** [comment]
Resolved with #561
